manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
+PackageLocale: en-US
+Publisher: Salesforce
+PackageName: sfdx-cli
+License: Copyright of Salesforce Technologies, All Rights Reserved

```

```suggestion
License: Proprietary
```
A copyright isn't a license.